Code2Gether-Discord-Bot.Tests/Fakes/FakeProjectManager.cs

```diff
@@ -1,66 +1,17 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using Code2Gether_Discord_Bot.Library.Models;
+﻿using Code2Gether_Discord_Bot.Library.Models;
 using Code2Gether_Discord_Bot.Library.Models.Repositories.ProjectRepository;
-using Discord;
 
 namespace Code2Gether_Discord_Bot.Tests.Fakes
 {
-    class FakeProjectManager : IProjectManager
+    /// <summary>
+    /// Represents a fake <see cref="ProjectManager"/>. This class cannot be inherited.
+    /// </summary>
+    internal sealed class FakeProjectManager : ProjectManager
```

Comment:
This object does not need to be faked. It should be tested instead. The underlying repository is faked.

Reply:
Thank for your review!
I made this class because... 

> I thought it'd be a good idea to work only with fake objects in the tests.